packages/app-form-builder/src/admin/components/FormEditor/Tabs/EditTab/EditTab.tsx

```diff
@@ -188,26 +193,42 @@ export const EditTab: React.FC = () => {
                                 </div>
                                 <Horizontal
                                     onDrop={item => {
-                                        handleStepMove(item, formStep);
+                                        handleStepMove({
+                                            target: {
+                                                containerId: item?.container?.id || "",
+                                                position: item.pos
+                                            },
+                                            destination: {
+                                                containerId: formStep.id
+                                            }
+                                        });
                                         return undefined;
                                     }}
                                     isVisible={item => {
                                         return (
                                             item.ui === "step" &&
-                                            renderTopDropZone(item?.formStep?.id, formStep.id)
+                                            renderTopDropZone(item?.container?.id, formStep.id)
                                         );
                                     }}
                                 />
                                 <Horizontal
                                     last
                                     onDrop={item => {
-                                        handleStepMove(item, formStep);
+                                        handleStepMove({
+                                            target: {
+                                                containerId: item?.container?.id || "",
+                                                position: item.pos
+                                            },
+                                            destination: {
+                                                containerId: formStep.id
+                                            }
+                                        });
                                         return undefined;
                                     }}
                                     isVisible={item => {
                                         return (
                                             item.ui === "step" &&
-                                            renderBottomDropZone(item?.formStep?.id, formStep.id)
+                                            renderBottomDropZone(item?.container?.id, formStep.id)
```

Comment:
`renderBottomDropZone(item.container?.id, formStep.id)` also works. In other words, no need to have `item?.`.

Ultimately, let's try to use `?.` only when absolutely needed. 

So, let's check this file if there are `?.` that can be removed.

Reply:
You are right, we don't need to use `item?.` it would never be `undefined` or `null`